examples/script/concolic.py

```diff
@@ -14,7 +14,10 @@
 '''
 
 import sys
-import Queue
+try:
+    import Queue
+except:
+    import queue as Queue
```

Comment:
__import queue__  # works on both Python 2 and Python 3 if we have done __pip install future__ as discussed at http://python-future.org/standard_library_imports.html

All remaining instances of __Queue__ in this file must be changed to __queue__.